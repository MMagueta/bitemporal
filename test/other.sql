\set QUIET 'on'

delete from s where id like 'other-%';

\echo 'other-01 - [empty]'
insert into s(id, value, valid_period) values ('other-01', 1, '[0,5)');
call save_s('other-01', 0, 'empty');
