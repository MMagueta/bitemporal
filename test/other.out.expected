other-01 - [empty]